### ISSUE TITLE
BridgeException (timeout) during long-running async JS call

### ISSUE THREAD

**Issue by austincondict-AngelOak** [extremeheat/JSPyBridge#57]
TLDR: Is there any way to dynamically set the call timeout (i.e. `REQ_TIMEOUT` in Bridge.js) to a different value, perhaps when instantiating a subclass of `PyClass`?

Here's my use case:
* I've created a PDF toolkit in Python (using `PyPDF2`)
* I've created a Javascript class (which extends `PyClass`) to wrap the Python class exactly as instructed in your docs/examples
* Functionally, everything works perfectly on a small scale, but I'm currently using my JS class on a Node server that has to deal with merging & obtaining page counts for very large PDF files
* The long-running async calls to my PDF toolkit class for these large files often incur a `BridgeException` due to the Python API call timing out (the call isn't actually hung, it's just taking longer than 100 seconds)

All problems are fixed if I increase `REQ_TIMEOUT` to something like `300_000` in my local node_modules, but I would love if I didn't have to change this value manually every time I provision a server w/ my project code. 

Is this possible? Or can you suggest any alternative workarounds for long-running async calls from Node?

**Comment by jc-roman** [comment]
@austincondict-AngelOak, where you able to find a way to dynamically set the timeout? Modifying the source file in node_modules isn't practical in a CI environment where an automated replace would be needed after running `npm install`. Any ideas for a CI friendly solution?
